action_server/src/robocorp/action_server/cli.py

```diff
@@ -114,10 +115,10 @@ def _create_parser():
         help="Expose the server to the world",
     )
     start_parser.add_argument(
-        "--expose-session",
-        dest="expose_session",
-        help="Restart action server with existing expose session",
-        default=None,
+        "--expose-y",
```

Comment:
Naming is hard... could this be:
`--expose-allow-reuse` or something.

Reply:
Renamed to `--expose-allow-reuse` 👍